docs/sdk-and-tools/sdk-py/mxpy-cli.md

```diff
@@ -329,3 +329,83 @@ mxpy tx new --pem ~/multiversx-sdk/testwallets/latest/users/alice.pem --recall-n
 That's it! As easy as that. We sent a transaction from Alice to Bob. We choose the receiver of our transaction using the `--receiver` argument and set the gas limit to `50000` because that is the gas cost of a simple move balance transaction. Notice we used the `--value` argument to pass the value that we want to transfer but we passed in the denomintated value. We transferred 1 eGLD (1 * 10^18). We then specify the proxy and the chain ID for the network we want to send our transaction to and use the `--send` argument to broadcast it.
 
 In case you want to save the transaction you can also provide the `--outfile` argument and a `json` file containing the transaction will be saved at the specified location. If you just want to prepare the transaction without broadcasting it simply remove the `--send` argument.
+
+## Using the Ledger hardware wallet
+
+You can sign any transaction (regular transfers, smart contract deployments and calls) using a Ledger hardware wallet by leveraging the `--ledger` command-line argument.
+
+First, connect your device to the computer, unlock it and open the MultiversX Ledger app.
+
+Then, you can perform a trivial connectivty check by running:
+
+```sh
+mxpy ledger version
+```
+
+The output should look like this:
+
+```sh
+MultiversX App version: ...
+```
+
+Another trivial check is to ask the device for the (first 10) MultiversX addresses it manages:
+
+```sh
+mxpy ledger addresses
+```
+
+The output should look like this:
+
+```sh
+account index = 0 | address index = 0 | address: erd1...
+account index = 0 | address index = 1 | address: erd1...
+account index = 0 | address index = 2 | address: erd1...
+...
+```
+
+Now let's sign and broadcast a transaction (EGLD transfer):
+
+```sh
+mxpy tx new --proxy https://devnet2-gateway.multiversx.com --recall-nonce \
```

Comment:
Let's use `devnet` instead of `devnet2`. For the other commands as well.

Reply:
Indeed, fixed.